pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -71,66 +71,36 @@ read_10x_files <- function(config, input_dir) {
       paste(sample_fpaths, collapse = " - ")
     )
 
-    counts <- Seurat::Read10X(sample_dir, gene.column = 1)
+    annotations <- read_10x_annotations(annot_fpath, sample)
+    counts <- Seurat::Read10X(sample_dir, gene.column = annotations[["gene_column"]], unique.features = TRUE)
 
     if (is(counts, "list")) {
       slot <- "Gene Expression"
-      # questionable: grab first slot if no gene expression
+      # questionable: grab first slot if no slot named gene expression
       if (!slot %in% names(counts)) slot <- names(counts)[1]
       counts <- counts[[slot]]
     }
```

Comment:
add comment about discarding elements in list. why we discard the rest of the matrices